builds/papas-build.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+#---Create stampfile to enable our jenkins to purge old builds------------------------------
+touch $WORKSPACE/controlfile
+
+#---Set up environment----------------------------------------------------------------------
+cd $WORKSPACE/papas
+source $WORKSPACE/papas/init.sh
+
+#---Prepare folder to store build artifacts-------------------------------------------------
+mkdir $WORKSPACE/papas/build
+cd $WORKSPACE/papas/build
+
+#---Run installation------------------------------------------------------------------------
+cmake -DCMAKE_INSTALL_PREFIX=../install ..
+make
```

Comment:
Might as well take advantage of our nice build machines and add `-j 16` :)

Reply:
Well spotted!